net-vpn/i2p/i2p-2.1.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="test"
+
+inherit java-pkg-2 java-ant-2 systemd
+
+DESCRIPTION="A privacy-centric, anonymous network"
+HOMEPAGE="https://geti2p.net"
+SRC_URI="https://files.i2p-projekt.de/${PV}/i2psource_${PV}.tar.bz2"
+
+LICENSE="Apache-2.0 Artistic BSD CC-BY-2.5 CC-BY-3.0 CC-BY-SA-3.0 EPL-1.0 GPL-2 GPL-3 LGPL-2.1 LGPL-3 MIT public-domain WTFPL-2"
+SLOT="0"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="nls"
+
+CP_DEPEND="
+	net-libs/nativebiginteger:0
+"
+
+DEPEND="${CP_DEPEND}
+	>=virtual/jdk-1.8:*
+	nls? ( >=sys-devel/gettext-0.19 )
+	test? (
+		dev-java/ant-junit4:0
+		dev-java/junit:4
+		dev-java/hamcrest:0
+		dev-java/mockito:4
+	)
+"
+
+RDEPEND="${CP_DEPEND}
+	acct-user/i2p
+	acct-group/i2p
+	>=virtual/jre-1.8:*
+"
+
+EANT_BUILD_TARGET="preppkg-base"
+# let ant expand gentoo.classpath itself
+EANT_EXTRA_ARGS='-Djavac.classpath=${gentoo.classpath}'
+# no scala as depending on antlib.xml not installed by dev-lang/scala
+EANT_TEST_TARGET="junit.test"
+EANT_TEST_GENTOO_CLASSPATH="junit-4,hamcrest,mockito-4"
+JAVA_ANT_ENCODING="UTF-8"
+
+DOCS=(README.md history.txt)
+
+src_prepare() {
```

Comment:
[java-pkg_clean](https://wiki.gentoo.org/wiki/Java_Developer_Guide#Removing_bundled_classes_and_jars), see [Replacing removed bundled jar files with system](https://wiki.gentoo.org/wiki/Java_Developer_Guide#Replacing%20removed%20bundled%20jars)
It looks like they're bundling a whole `tomcat` server, but needs to be checked more in detail.  Emerging `tomcat:9` and looking into `qlist tomcat` might help. Also `pkgdiff` for the questionable jar files.